75 nuxt help screens

sandragjacinto opened the pull request:
PR to merge help modal to new structure

sandragjacinto committed:
adds help modal changes to nuxt structure

sandragjacinto committed:
adds help modal changes to nuxt structure

sandragjacinto committed:
adds help modal changes to nuxt structure

sandragjacinto committed:
adds help modal changes to nuxt structure

shawnlauzon reviewed (changes requested):
I pasted my userid on step 3 of the help and it doesn't do anything (no response, not failure)

sandragjacinto committed:
adds checkProfile

sandragjacinto committed:
adds checkProfile

shawnlauzon committed:
Better logged in / logged out handling on navbar

shawnlauzon committed:
Change navbar based on logged in / logged out

sandragjacinto committed:
Merge pull request #128 from Volentix/126-loggedout-handling

Better logged in / logged out handling on navbar

shawnlauzon committed:
Short-term fix for token no longer on server

When the token is no longer accepted, we need a better way of
clearing it. This issue adds some code to help clean things, but
it needs to be better written.

sandragjacinto committed:
Merge pull request #130 from Volentix/129-unauthorized-error-after-clearing-server

Short-term fix for token no longer on server

sandragjacinto committed:
trying to fix error from createProfile

shawnlauzon committed:
Fix onboarding problem forum_user_id does not exist

shawnlauzon committed:
Log in link highlighted should only be highlighted on root

shawnlauzon committed:
Fix errors when displaying leaderboard unauthenticated

sandragjacinto committed:
handles conflicts
